### PR TITLE
feature (Numbers): Provide elegant functions for Active and Available

### PR DESCRIPTION
**Pull request by JPPortier**

### CODE REVIEW

File: client/src/main/com/sinch/sdk/domains/numbers/api/v1/adapters/NumbersService.java [hunk 1]
```diff
@@ -98,6 +108,50 @@ public WebHooksService webhooks() {
...
+   *
+   * @param phoneNumber Phone number
+   * @return Available number information
+   * @since 1.2
+   */
+  public AvailableNumber checkAvailability(String phoneNumber) throws ApiException {
+    return available().checkAvailability(phoneNumber);
+  }
+
+  public ActiveNumber rent(String phoneNumber, AvailableNumberRentRequest parameters)
```
Comment (asein-sinch) [inline client/src/main/com/sinch/sdk/domains/numbers/api/v1/adapters/NumbersService.java:120]: Only the `checkAvailability` method gets to have a javadoc

File: sample-app/src/main/java/com/sinch/sample/numbers/Get.java [hunk 2]
```diff
@@ -22,7 +22,7 @@ public static void main(String[] args) {
 
   public void run() {
 
-    ActiveNumberService service = client.numbers().v1().active();
+    NumbersService service = client.numbers().v1();
```
Comment (asein-sinch) [inline sample-app/src/main/java/com/sinch/sample/numbers/Get.java:25]: In the end, it was a good thing to name the variable just `service` 👍

**Review by asein-sinch — COMMENTED** [review]
This looks great! Kudos for the lightning fast implementation!

File: client/src/main/com/sinch/sdk/domains/numbers/api/v1/adapters/NumbersService.java (hunk 1 above)
Comment (JPPortier) [reply client/src/main/com/sinch/sdk/domains/numbers/api/v1/adapters/NumbersService.java:120]: Removed javadoc here: we are in "adapters" it do not have to be documented (and is not)

**Pull request merged**